modules/core/src/smithy4s/schema/Schema.scala

```diff
@@ -354,8 +354,9 @@ object Schema {
   private object OptionDefaultVisitor extends SchemaVisitor.Default[Option] {
```

Comment:
Note that I was not able to create any test cases that indicated an implementation for refinements would do anything here. Every case I could think of would still bypass calling the refinement case. The `DefaultValueSchemaVisitor` did need to be updated however.